src/build/render/components/bits.js

```diff
@@ -1,16 +1,17 @@
 const commonTags = require("common-tags");
+const R = require("ramda");
 
 const DISCORD_URL = "https://discord.reclaimers.net";
 const REPO_URL = "https://github.com/Sigmmma/c20";
-const MAX_DETAILS_LIST = 8;
+const DEFAULT_OPEN_THRESHOLD = 8;
 
 const breakTagName = (tagName) => tagName.split("_").join("_<wbr>");
 
 //converts a title into a URL- or ID-friendly slug
 const slugify = (title) => title
   .toLowerCase()
   .replace(/[']/g, "")
-  .replace(/[^a-z0-9]/g, " ")
+  .replace(/[^\p{L}0-9]/gu, " ")
```

Comment:
This change is to support any unicode letters rather than just a-z in "slugified" headings, e.g. converting "This is a heading" to "this-is-a-heading". This is used when generating IDs for heading elements on the page so we can do `#anchor-links-like-this`.